app/multitenant/dynamo_collector.go

```diff
@@ -49,19 +50,33 @@ var (
 		Name:      "dynamo_consumed_capacity",
 		Help:      "The capacity units consumed by operation.",
 	}, []string{"method"})
-	dynamoReportSize = prometheus.NewCounterVec(prometheus.CounterOpts{
+	dynamoValueSize = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace: "scope",
-		Name:      "dynamo_report_size_bytes",
-		Help:      "Size of reports read / written from dynamodb.",
+		Name:      "dynamo_value_size_bytes",
+		Help:      "Size of data read / written from dynamodb.",
 	}, []string{"method"})
+
+	reportSize = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "scope",
+		Name:      "report_size_bytes",
+		Help:      "Size of reports written to s3.",
```

Comment:
The name and help don't match.. Either it's just the size of reports written to s3, or it's the size of all reports (s3 or no)